packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -421,6 +421,11 @@ class AndroidCameraCameraX extends CameraPlatform {
     assert(cameraSelector != null);
     assert(processCameraProvider != null);
 
+    if (recording != null) {
+      // There is currently an active recording, so do not start a new one.
```

Comment:
Can you add a statement to the doc for this method that this is the expected behavior for trying to start a new recording?

Reply:
Added a note that we silently return without doing anything if there is an active recording.